packages/frontend/src/js/app/components/TownForm/TownFormPanelLocation.vue

```diff
@@ -86,7 +83,12 @@ export default {
             const longitude = this.input.coordinates[1];
             try {
                 const { towns } = await findNearby(latitude, longitude);
+                const { closedTowns } = await findClosedNearby(
+                    latitude,
+                    longitude
+                );
                 this.nearbyShantytowns = towns;
+                this.$emit("shareClosedTowns", closedTowns);
```

Comment:
Je ferais un premier emit avec un tableau vide, au-dessus du try, pour reset la liste avant d'en demander une nouvelle. L'intérêt étant de gérer le cas où l'API échoue à répondre et que du coup ta liste de sites fermés proche ne s'actualise pas.

Tu pourrais le faire dans le `catch`, mais en vérité à partir du moment où les coordonnées changent, théoriquement la liste devient caduque donc autant la reset le plus vite possible.

Rajouter donc un `this.$emit("shareClosedTowns", []);`.